<?xml version='1.0' encoding='utf-8'?>
<ui version="4.0">
  <class>vfx_file_browser</class>
  <widget class="QMainWindow" name="vfx_file_browser">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>vfx_file_browser</string>
    </property>
    <widget class="QLabel" name="pathLabel">
      <property name="text">
        <string>경로:</string>
      </property>
      <property name="geometry">
        <rect>
          <x>10</x>
          <y>10</y>
          <width>50</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QLineEdit" name="pathInput">
      <property name="text">
        <string>/home/m83/chulho</string>
      </property>
      <property name="geometry">
        <rect>
          <x>70</x>
          <y>10</y>
          <width>550</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QPushButton" name="browseButton">
      <property name="text">
        <string>찾아보기...</string>
      </property>
      <property name="geometry">
        <rect>
          <x>630</x>
          <y>10</y>
          <width>80</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QPushButton" name="refreshButton">
      <property name="text">
        <string>새로고침</string>
      </property>
      <property name="geometry">
        <rect>
          <x>720</x>
          <y>10</y>
          <width>70</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QLabel" name="filterLabel">
      <property name="text">
        <string>필터:</string>
      </property>
      <property name="geometry">
        <rect>
          <x>10</x>
          <y>45</y>
          <width>50</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QComboBox" name="filterCombo">
      <property name="geometry">
        <rect>
          <x>70</x>
          <y>45</y>
          <width>150</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QCheckBox" name="showHiddenCheck">
      <property name="text">
        <string>숨김 파일 표시</string>
      </property>
      <property name="geometry">
        <rect>
          <x>230</x>
          <y>45</y>
          <width>120</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QCheckBox" name="recursiveCheck">
      <property name="text">
        <string>하위 폴더 포함</string>
      </property>
      <property name="geometry">
        <rect>
          <x>360</x>
          <y>45</y>
          <width>120</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QListWidget" name="fileList">
      <property name="geometry">
        <rect>
          <x>10</x>
          <y>80</y>
          <width>380</width>
          <height>450</height>
        </rect>
      </property>
    </widget>
    <widget class="QLabel" name="previewLabel">
      <property name="text">
        <string>미리보기</string>
      </property>
      <property name="geometry">
        <rect>
          <x>400</x>
          <y>80</y>
          <width>390</width>
          <height>30</height>
        </rect>
      </property>
    </widget>
    <widget class="QLabel" name="previewImage">
      <property name="text">
        <string>(이미지 미리보기 영역)</string>
      </property>
      <property name="geometry">
        <rect>
          <x>400</x>
          <y>115</y>
          <width>390</width>
          <height>300</height>
        </rect>
      </property>
    </widget>
    <widget class="QTextEdit" name="fileInfo">
      <property name="geometry">
        <rect>
          <x>400</x>
          <y>425</y>
          <width>390</width>
          <height>105</height>
        </rect>
      </property>
    </widget>
    <widget class="QLabel" name="statusLabel">
      <property name="text">
        <string>준비</string>
      </property>
      <property name="geometry">
        <rect>
          <x>10</x>
          <y>540</y>
          <width>600</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
        <rect>
          <x>620</x>
          <y>540</y>
          <width>170</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QPushButton" name="openButton">
      <property name="text">
        <string>열기</string>
      </property>
      <property name="geometry">
        <rect>
          <x>500</x>
          <y>540</y>
          <width>90</width>
          <height>40</height>
        </rect>
      </property>
    </widget>
    <widget class="QPushButton" name="importButton">
      <property name="text">
        <string>임포트</string>
      </property>
      <property name="geometry">
        <rect>
          <x>600</x>
          <y>540</y>
          <width>90</width>
          <height>40</height>
        </rect>
      </property>
    </widget>
    <widget class="QPushButton" name="closeButton">
      <property name="text">
        <string>닫기</string>
      </property>
      <property name="geometry">
        <rect>
          <x>700</x>
          <y>540</y>
          <width>90</width>
          <height>40</height>
        </rect>
      </property>
    </widget>
  </widget>
</ui>
